### DISCUSSION TITLE
Is it possible to show the files changed in a PR?

### DISCUSSION THREAD

**Discussion by LemonBreezes** [magit/forge#631] [Support] [answered]
Hi. Is it possible to add a section to `magit-forge` to show the file change hunks? It would be really nice to see what changes merging a PR would do without hunting through the commits.

**Comment by LemonBreezes** [comment]
Okay, I just realized if I do a 3 dots diff, `master...upstream/pr/7663`, it shows me the diff I want, which is the diff since the branches diverged.

**Reply by tarsius** [reply, marked as answer]
You can avoid having to enter that range at a prompt, by moving the cursor onto the pull-request in the status buffer and using "dwim diff" (`d d`) there. Also works on the `Commits` section when viewing the pull-request in a separate buffer.
